tools/packaging/debian/dragonfly.logrotate

```diff
@@ -1,7 +1,26 @@
-/var/log/dragonfly/dragonfly* {
-        weekly
+# installed by debhelper by convention into /etc/logrotate.d/
+
+/var/log/dragonfly/dragonfly*.log {
+        daily
         missingok
-        rotate 12
+
         compress
+        compresscmd zstd
```

Comment:
In older versions of Debian `zstd` does not appear to be installed, so might need to add it as a dependency to the package.

Reply:
Thanks! Do you know how to declare dependencies in debian package?

Reply:
I assume it's this: https://github.com/dragonflydb/dragonfly/blob/88852ccdd536d1b8b7141e0b9411837cb92f8f6e/tools/packaging/debian/control#L10

Reply:
Thanks, will update! btw, I never asked you. Why did you decide to implement the slowlog feature?  Are you using Dragonfly or just for fun? Feel free to DM me on discord if you wish :)

Reply:
answered on Discord!

Reply:
fixed dependency issue.